run/VEGPARM.TBL

```diff
@@ -102,27 +102,27 @@ NATURAL
 CROP
 12
 LCZ_1
-31
+51
 LCZ_2
-32
+52
 LCZ_3
-33
+53
 LCZ_4
-34
+54
 LCZ_5
-35
+55
 LCZ_6
-36
+56
 LCZ_7
-37
+57
 LCZ_8
-38
+58
 LCZ_9
-39
+59
 LCZ_10
-40
+60
 LCZ_11
-41
+61
```

Comment:
These changes seem to indicate some inconsistency in branches because they should already be in